src/functions.php

```diff
@@ -395,3 +395,11 @@ function bin()
         };
     }
 }
+
+/*
+ * Use `eval(sh)` because `eval(\Psy\sh());` is too long
+ */
+$constant = (new Configuration())->getDebugConstant();
```

Comment:
Using `Configuration` like this is problematic. There's no guarantee that the configuration you get instantiating it this way is actually the one in use in the eventual PsySH session (for example, [if the config file is overridden via the `--config` flag](https://github.com/bobthecow/psysh/blob/master/src/Configuration.php#L173)).

Even if it does end up with the right config file, adding this in `src/functions.php` (or any other non-autoloaded PHP file) isn't great. It's run on every request to every webserver and every CLI app with PsySH installed as a dependency, regardless of whether someone intends to use it or not.

The current approach is almost a no-op until someone actually calls `Psy\sh()` which kicks off a bunch of autoloading and object instantiating. I don't think we can justify the overhead just to save people a few characters, _especially_ since the default is just `eval(Psy\sh())` to `eval(Psy\sh)`.